screens/Home/MyVcs/HistoryTab.tsx

```diff
@@ -27,7 +28,7 @@ export const HistoryTab: React.FC<HistoryTabProps> = (props) => {
         </ListItem.Title>
       </ListItem.Content>
       <Modal
-        headerLabel={props.vcKey.split(':')[2]}
+        headerLabel={props.vcKey.split(':')[5]}
```

Comment:
@tilak-puli this will be impacted after INJI-344 changes
here we are showing UIN on header

Reply:
let me check this once.